dev/gateway

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
There is already a `dev/run-gateway` that looks nearly identical to this

Reply:
oh. I will merge them